src/main/java/lesson9Text/TextAnalyzer.java

```diff
@@ -0,0 +1,23 @@
+package lesson9Text;
+
+public class TextAnalyzer {
+
+    public static String analyzeText(String text){
+        if (text.equals("")){
+            return "this is an empty string";
+        }
+        return String.format("length: %s\nis palindrome: %b", getTextLength(text), isPalindrome(text));
+    }
+
+    public static boolean isPalindrome(String text) {
+        text = text.replaceAll("[^a-zA-Z0-9]","").toLowerCase();
```

Comment:
Dlaczego usuwasz wszystkie litery i cyfry z tekstu?

Reply:
po to by taki tekst był uznany jako palindrom:
"Niagara. O roar again!"